### PR TITLE
account_activations

### PR DESCRIPTION
**Pull request by ngyihkai**
Ready

### CODE REVIEW

**Comment by ngyihkai** [comment]
Ready

File: app/assets/stylesheets/account_activations.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the AccountActivations controller here.
+// They will automatically be included in application.css.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/account_activations.scss:2]: delete this file because not use

File: app/helpers/account_activations_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
+end
```
Comment (ngocvt-0484) [inline app/helpers/account_activations_helper.rb:2]: same. delete

File: app/mailers/user_mailer.rb [hunk 3]
```diff
@@ -5,10 +5,10 @@ class UserMailer < ApplicationMailer
   #
   #   en.user_mailer.account_activation.subject
   #
-  # def account_activation(user)
```
Comment (ngocvt-0484) [inline app/mailers/user_mailer.rb:8]: delete this line. not comment

Comment (ngyihkai) [reply app/mailers/user_mailer.rb:8]: I already remove the comment and need to use this code to activate the account for the user

**Comment by ngyihkai** [comment]
Ready

**Review by anlvq-1058 — COMMENTED** [review]
merged

**Pull request merged**